pandas/core/groupby/groupby.py

```diff
@@ -403,15 +403,23 @@ class providing the base-class of operations.
 f : function, str
     Function to apply to each group. See the Notes section below for requirements.
 
-    Can also accept a Numba JIT function with
-    ``engine='numba'`` specified.
+    Accepted inputs are:
+
+    - String function name
+    - Standard Python function
+    - User defined functions IE ``lambda x: x.sum()``
```

Comment:
These are the same - I think we can just say `Python function`

Reply:
Alright, hopefully this better: 9223b59e6aac77dc0a12eb8585ce1e8e0c9f52d4